cabal.project

```diff
@@ -33,3 +33,8 @@ source-repository-package
   type: git
   location: https://github.com/tclem/proto-lens-jsonpb
   tag: e4d10b77f57ee25beb759a33e63e2061420d3dc2
+
+source-repository-package
+  type: git
+  location: https://github.com/antitypical/fused-syntax.git
+  tag: 5b7512db962d5b3f973002615b8bc86ab074d5aa
```

Comment:
We’ll have to do a Hackage release of this eventually, but this is fine for now.

Reply:
Yep. I’m mostly held up debating whether to keep or 🔥 the `Analysis` class it has.

I’m also definitely going to rename it.